bin/plot_acorrs.py

```diff
@@ -63,7 +68,17 @@ def main(data_dct):
     print("  Percent of steady correlated pexecs: %s" % percent)
 
 
-def analyse_correlations(data, key, steady_idxs):
+    print("  Skew threshold: +/-%s" % SKEW_THRESHOLD)
+    print("  Kurtosis threshold: +/-%s" % KURT_THRESHOLD)
+    print("  Total num steady pexecs approximately normal: %s" % total_num_normal)
+    if total_num_normal:
+        percent = float(total_num_normal) / total_num_steady * 100
+    else:
+        percent = "N/A"
+    print("  Percent of steady pexecs also normal: %s" % percent)
```

Comment:
I think the "also" here is a bit confusing, maybe stick with "approximately", as above?